include/polycap-photon.h

```diff
@@ -24,42 +30,92 @@
 extern "C" {
 #endif
 
+/** Struct describing a 3 dimensional vector where x and y are horizontal and vertical directions compared to the polycapillary respectively, and z is the direction along the polycapillary length
+ */
 typedef struct {
-	double x;
-	double y;
-	double z;
+	double x; ///< horizontal X-axis vector contribution
+	double y; ///< vertical Y-axis vector contribution
+	double z; ///< Z-axis (perpendicular to X and Y) vector contribution
 } polycap_vector3;
 
+
+struct _polycap_leak;
```

Comment:
Best to remove all references to polycap_leak if it is not going to be part of the public API: no need to confuse the reader! :smile: